components/nodemanager-service/pgdb/node-managers.go

```diff
@@ -339,24 +339,15 @@ func validateNodeManagerFilters(filters []*common.Filter) error {
 		switch filter.Key {
 		case "manager_type":
 			for _, item := range filter.Values {
-				if !isValidManagerType(item) {
-					return &errorutils.InvalidError{Msg: fmt.Sprintf("Invalid manager type filter: %s. manager_type must be one of the following: 'aws-ec2', 'aws-api', 'azure-api'", item)}
+				if !isValidNodeManagerType(item) {
+					return &errorutils.InvalidError{Msg: fmt.Sprintf("Invalid manager type filter: %s. manager_type must be one of the following: 'aws-ec2', 'aws-api', 'azure-api', 'chef', '', 'azure-vm', 'gcp-api' ", item)}
```

Comment:
So, the manager type can still be empty?

Reply:
no new ones should be empty, but we need to keep this around for the old nodes. they should get updated once a new run/scan comes thru but if there are no new runs/scans for the node it will be stuck in the empty manager state